lib/tasks/link-audit.rake

```diff
@@ -0,0 +1,22 @@
+desc "Determine if all websites have links"
+task link_audit: :environment do
+  Redirection.find_each do |redirection|
+    url = URI(redirection.url)
```

Comment:
Yeah I kind of use those interchangeably. I guess URI is more generic in that it's _some_ form of identifier, and URLs are specifically for the web. So I am technically dealing with URLs even if the Net::HTTP requires a URI. Anyway, yeah I'll change it.